### PR TITLE
Small doc improvements on Bundles

### PR DESCRIPTION
**Pull request by educhastenier**

### CODE REVIEW

File: md/wildfly-bundle.md [hunk 1]
```diff
@@ -164,19 +170,38 @@ WildFly can be shut down by executing the following script:
 
 You can also press Ctrl + C.
 
+<a id="post-install" />
```
Comment (educhastenier) [inline md/wildfly-bundle.md]: Why add an new **anchor** if it is not referenced?

Comment (NathalieC) [reply md/wildfly-bundle.md]: I haven't added it. 
If you are sure it's not used, I can remove it.
Can you test?
Let me know.

Comment (educhastenier) [reply md/wildfly-bundle.md]: According to https://github.com/bonitasoft/bonita-doc/pull/190/commits/91aa7934df91e601dba83c00b1088d181f3f2bf3, you did it (maybe not on purpose).
I'll remove it.

File: md/BonitaBPM_platform_setup.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 # Platform configuration
 
-
 In this section you will learn what the Platform setup tool is and how to use it to initialize and configure the Bonita BPM Platform.
 
+:::info
+**Note:** First thing to know is that for all basic installation and start scenarios, you don't need to use the setup tool. 
+Run `start-bonita.bat`(for Windows) or `sart-bonita.sh` (for Unix) to initialize, configure and start the bundle.
+Use the setup tool to update the configuration of Bonita BPM Platform, update the license file, or manage advanced installation and start scenarios.
+:::
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md:9]: :+1:  Nice introduction

File: md/BonitaBPM_platform_setup.md [hunk 3]
```diff
@@ -34,22 +38,27 @@ It is composed of the following items:
...
-  It pushes the configuration from the `platform_conf/current` folder into the database of an already initialized Bonita BPM Platform
+  It gets the current configuration of Bonita BPM Platform in the database and saves it in the `platform_conf/current` folder.  
 
-* Configure a Bonita BPM Tomcat / WildFly bundle: `configure`
+* Updates current configuration: `push`
 
-  If run from inside a Bonita BPM bundle, it configures the Application Server environment, preventing the need to configure all Tomcat / WildFly configuration files manually.
+  It pushes the modified configuration from the `platform_conf/current` folder into the database of an already initialized Bonita BPM Platform
+
+**Note:** Those two actions must be used to update the configuration of Bonita BPM Platform and update the license file.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: Those **last** two actions

File: md/BonitaBPM_platform_setup.md [hunk 4]
```diff
@@ -34,22 +38,27 @@ It is composed of the following items:
...
+A `setup pull`action must be performed before et `setup push`action to avoid database inconsistency.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: ~~et~~ => a

File: md/BonitaBPM_platform_setup.md [hunk 5]
```diff
@@ -58,13 +67,13 @@ This tool has 4 functions:
 
 Before running it, the tool must be configured to point to the database of the Bonita BPM Platform.
 
-To do so, after you have created the database, modify the tool `database.properties` file:  
+To do so, after you have created the database and customized it so it works with Bonita BPM, modify the tool `database.properties` file:
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: remove ~~tool~~ word

File: md/BonitaBPM_platform_setup.md [hunk 6]
```diff
@@ -82,50 +91,26 @@ Remember that in the [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md)
...
 <a id="run_bundle_configure" />
 
-## Run the Bonita BPM bundle configuration (Tomcat 7 / WildFly 10)
+## Configure the server (Tomcat 7 / WildFly 10)
 
 ::: info
 In the [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md) bundles, configuration is done automatically when calling `start-bonita.sh / .bat`
 :::
 
 1. Update the file `database.properties` with the database properties matching your environment (db vendor name, server name, server port, connection username and password)
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: Why not say **Configure the Platform setup tool as explained above.** like for `init` command?

File: md/BonitaBPM_platform_setup.md [hunk 7]
```diff
@@ -82,50 +91,26 @@ Remember that in the [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md)
 
 To do so, proceed as follows:
 
-1. Create a new empty database.
-2. Configure the *Platform setup tool* as explained above.
-3. From the tool, run `setup.sh init` (Unix/Mac) or `setup.bat init` (Windows).
+   1. Create a new empty database.
+   2. Configure the *Platform setup tool* as explained above.
+   3. From the tool, run `setup.sh init` (Unix/Mac) or `setup.bat init` (Windows).
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: From the tool **folder**

File: md/BonitaBPM_platform_setup.md [hunk 8]
```diff
@@ -165,6 +150,31 @@ For WildFly:
...
+
+<a id="update_platform_conf" />
+
+## Update Bonita BPM Platform configuration or licenses
+
+If you want to modify the configuration or update a license in an already initialized Bonita BPM Platform, you must use the *Platform setup tool* as follows:
+
+   1. Stop Bonita BPM Platform (optional)
+   2. If not already done, update the `database.properties` of the *Platform setup tool* to point to the database used by Bonita BPM Platform.
+   3. Run `setup.sh pull` or `setup.bat pull` from the tool. It will get the current configuration and put it in the `platform_conf/current` folder.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: From the tool **folder**

File: md/BonitaBPM_platform_setup.md [hunk 9]
```diff
@@ -165,6 +150,31 @@ For WildFly:
...
+   4. Modify the configuration files inside the `platform_conf/current` folder according to your needs.
+   5. If you are updating your license file, put it inside `platform_conf/licenses`, along with the existing ones. If some retrieved license files are not valid anymore, you can remove them, to delete them from the database when pushed.
+   6. Run `setup.sh push` or `setup.bat push` from the tool.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: From the tool **folder**

File: md/wildfly-bundle.md [hunk 10]
```diff
@@ -46,18 +46,29 @@ The WildFly bundle is based on a standard WildFly installation with the followin
...
 
-
 ::: info
 **Note:** Beginning with version 7.3.0, Bonita BPM Platform configuration, including the license file, is stored in the same database as the Bonita BPM Engine data, namely in the `CONFIGURATION` table.  
-To initialize and update this configuration, a [*Platform setup tool*](BonitaBPM_platform_setup.md) is provided and embedded in Bonita BPM bundles. 
-It will be launched automatically when you start the WildFly bundle to initialize the database.
+The initialization of this configuration happens during start-bonita.bat (for Windows) or start.bonita.sh (for Linux) execution. To update this configuration, use the [*Platform setup tool*](BonitaBPM_platform_setup.md) embedded in Bonita BPM bundles.  
 :::
 
+### Get and install a license (Subscription editions only)
```
Comment (educhastenier) [inline md/wildfly-bundle.md:55]: Is it really necessary to copy all this **license** part?

File: md/tomcat-bundle.md [hunk 11]
```diff
@@ -62,20 +62,33 @@ The Tomcat bundle is based on a standard Tomcat installation with the following
 
 ::: info
 **Note:** Beginning with version 7.3.0, Bonita BPM Platform configuration, including the license file, is stored in the same database as the Bonita BPM Engine data, namely in the `CONFIGURATION` table.
-To initialize and update this configuration, a [*Platform setup tool*](BonitaBPM_platform_setup.md) is provided and embedded in Bonita BPM bundles.
+The initialization of this configuration happens during `start-bonita.bat` (for Windows) or `start.bonita.sh` (for Linux) execution.
+To update this configuration, use the [*Platform setup tool*](BonitaBPM_platform_setup.md) embedded in Bonita BPM bundles.
 :::
 
-
 ### Get and install a license (Subscription editions only)
```
Comment (educhastenier) [inline md/tomcat-bundle.md:69]: Is it really necessary to copy all this **license** part?

File: md/BonitaBPM_platform_setup.md (hunk 3 above)
Comment (NathalieC) [reply md/BonitaBPM_platform_setup.md]: done

File: md/BonitaBPM_platform_setup.md (hunk 4 above)
Comment (NathalieC) [reply md/BonitaBPM_platform_setup.md]: thx. done

File: md/BonitaBPM_platform_setup.md (hunk 6 above)
Comment (NathalieC) [reply md/BonitaBPM_platform_setup.md]: I simplified quite a bit so as to avoid repetitions.

File: md/BonitaBPM_platform_setup.md [hunk 12]
```diff
@@ -1,34 +1,32 @@
...
 # Platform configuration
 
-In this section you will learn what the Platform setup tool is and how to use it to initialize and configure the Bonita BPM Platform.
+In this section you will learn what the Platform setup tool is and how to use it to configure the Bonita BPM Platform, manage your licenses (for Subscription users), and for special use cases, initialize the bundle.
 
 :::info
 **Note:** First thing to know is that for all basic installation and start scenarios, you don't need to use the setup tool. 
-Run `start-bonita.bat`(for Windows) or `sart-bonita.sh` (for Unix) to initialize, configure and start the bundle.
-Use the setup tool to update the configuration of Bonita BPM Platform, update the license file, or manage advanced installation and start scenarios.
+We made it easy for you by creating `start-bonita.bat`(for Windows) or `sart-bonita.sh` (for Unix), a script that initializes, configures and starts the bundle, using the Platform setup tool.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: typo `start`

File: md/BonitaBPM_platform_setup.md [hunk 13]
```diff
@@ -1,34 +1,32 @@
 
 # Platform configuration
 
-In this section you will learn what the Platform setup tool is and how to use it to initialize and configure the Bonita BPM Platform.
+In this section you will learn what the Platform setup tool is and how to use it to configure the Bonita BPM Platform, manage your licenses (for Subscription users), and for special use cases, initialize the bundle.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: `for special use cases, initialize the bundle` => I don't understand what you mean...

File: md/BonitaBPM_platform_setup.md [hunk 14]
```diff
@@ -1,34 +1,32 @@
...
+## Platform setup tool overview and use
 
-The *Platform setup tool* handles the creation of the database schema and the configuration of Bonita BPM Platform.
-It also allows to configure a Tomcat or a WildFly bundle without the need for a fully manual configuration.  
-It is located in the [deploy bundle](deploy-bundle.md) and in both [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md) bundles.
-
-::: info
-In Tomcat and WildFly bundles, you can find the tool in the `setup` folder
-:::
+The *Platform setup tool* handles:
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md:15]: Good job :+1:  this paragraph is clearer like this.

File: md/BonitaBPM_platform_setup.md [hunk 15]
```diff
@@ -1,34 +1,32 @@
...
+It is located in both [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md) bundles, and also in the [deploy bundle](deploy-bundle.md). In Tomcat and WildFly bundles, you can find the tool in the `setup` folder.
 
 ### Structure
 
 It is composed of the following items:
 * `platform_conf/`
     * `initial/`: contains the default configuration of Bonita BPM Platform, that can be customized and will be pushed when the database is created.
     * `current/`: contains configuration files after a `pull` from the database is made.
-    * `licenses/`: (Subscriptions only) ==> must contain the license file to allow Bonita BPM Platform to start without error.
+    * `licenses/`: (Subscriptions only) contains the license file to allow Bonita BPM Platform to start without error.
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: I would rather say: **folder to put the license file** ...

File: md/BonitaBPM_platform_setup.md [hunk 16]
```diff
@@ -37,82 +35,72 @@ It is composed of the following items:
 
 ### Functions
 
-This tool has 4 functions:
-
-* Initializes the database: `init`
+The script `setup`comes with 4 options:
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: missing `space`

File: md/BonitaBPM_platform_setup.md [hunk 17]
```diff
@@ -37,82 +35,72 @@ It is composed of the following items:
...
-* Retrieves current configuration: `pull`
+  It creates the database tables and pushes the initial configuration in the database.  
+  This initial configuration is taken from the `platform_conf/initial` folder.  
+  It is run by the global script `start_bonita`, but you can also insert it in your own scripts.
 
-  It gets the current configuration of Bonita BPM Platform in the database and saves it in the `platform_conf/current` folder.  
+<a id="config_platform_conf" />
 
-* Updates current configuration: `push`
+* `configure`, to configure the server of a Bonita BPM Tomcat / WildFly bundle on the database:
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md]: "WildFly bundle ~~on~~ **to use** the **appropriate** database"

File: md/BonitaBPM_platform_setup.md [hunk 18]
```diff
@@ -37,82 +35,72 @@ It is composed of the following items:
...
-   2. Configure the *Platform setup tool* as explained above.
-   3. From the tool, run `setup.sh init` (Unix/Mac) or `setup.bat init` (Windows).
-
-This last step will connect to the database pointed to in `database.properties`, create the tables required for the platform to run, push all configuration from the `platform_conf/initial` folder and push the licenses from the licenses folder.
-
-That's it! You can now run Bonita BPM Platform on this database.
-
+<a id="configure_tool" />
 
-<a id="run_bundle_configure" />
```
Comment (educhastenier) [inline md/BonitaBPM_platform_setup.md:118]: you cannot remove **run_bundle_configure**. It is used by other pages.

File: md/BonitaBPM_platform_setup.md (hunk 12 above)
Comment (educhastenier) [reply md/BonitaBPM_platform_setup.md]: fixed

**Pull request merged**